openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -573,10 +574,12 @@ class Celebration(DeveloperErrorViewMixin, APIView):
         Body consists of the following fields:
```

Comment:
the behavior of the streak code and first section code is a bit different 
it seems like it would be nice for them to use the same endpoint/be in the same place, though not sure if this will open up the possibility for confusion if the celebrations work differently
(for instance, for the first section, it is simply letting us know that the user was celebrating, while for the streak, we are checking whether we should celebrate and updating the streak tracking)